File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.Ex_PB-JS-Conditional-Statements/P06_WorldSwimmingRecord.js
function worldSwimmingRecord(input) {
    let recordSecs = Number(input[0]);
    let distanceMets = Number(input[1]);
    let oneMetsTime = Number(input[2]);

    let currentTime = distanceMets * oneMetsTime + (Math.floor(distanceMets / 15)) * 12.5;

    if (currentTime < recordSecs) {
        console.log(`Yes, he succeeded! The new world record is ${currentTime.toFixed(2)} seconds.`);
    } else {
        console.log(`No, he failed! He was ${(currentTime - recordSecs).toFixed(2)} seconds slower.`);
    }
}

worldSwimmingRecord(["55555.67",
    "3017",
    "5.03"])

File: Java_Basic_Jan_2023_SolutionsOfProblems/P02Ex_ConditionalStatements/P06_WorldSwimmingRecord.java
package P02Ex_ConditionalStatements;

import java.util.Scanner;

public class P06_WorldSwimmingRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double recordSeconds = Double.parseDouble(scanner.nextLine());
        double distanceMeters = Double.parseDouble(scanner.nextLine());
        double timeSeconds1Meter = Double.parseDouble(scanner.nextLine());
        double time = distanceMeters * timeSeconds1Meter + Math.floor(distanceMeters / 15) * 12.5;

        if (time < recordSeconds)
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", time);
        else
            System.out.printf("No, he failed! He was %.2f seconds slower.", time - recordSeconds);
    }
}
